test/xml/CatLibrary/Cat`2.xml

```diff
@@ -93,9 +93,9 @@
     <remarks>
```

Comment:
@TianqiZhang:  I had some difficulty regenerating the XML using the `gen-xml.ps1` script in the `test` folder.  Running it introduced a lot of subtle formatting changes which upset the diff tool.   I suspect that it has to do with the .NET SDK versions.

Reply:
@jsquire thank you for your contribution! The diff is probably caused by the fact that sometimes people modified the xml directly, instead of changing the test assembly source code. I tested `gen-xml.ps1` locally and saw many modified lines in ``Cat`2.xml``. I'll fix the xml diff first so it won't be confusing.